Src/GoogleApis.Tests/Apis/Download/MediaDownloaderTest.cs

```diff
@@ -59,6 +60,9 @@ private class MultipleChunksMessageHandler : CountableMessageHandler
             /// <summary>Gets or sets the chunk size to send.</summary>
             public int ChunkSize { get; set; }
 
+            /// <summary>Sets the status code to return for simulating errors.</summary>

```

Comment:
s/Sets/Gets or sets
I would change it to:
Gets or sets the status code in case we simulate an error.